lib/cog/command/pipeline/executor.ex

```diff
@@ -387,6 +390,25 @@ defmodule Cog.Command.Pipeline.Executor do
   defp default_template(_),
     do: "raw"
 
+  defp render_templates(command_output, command, adapter) do
+    rendered = Enum.reduce_while(command_output, [], fn({data, template}, acc) ->
+      try do
+        {:cont, [render_template(command, adapter, template, data) | acc]}
```

Comment:
Just a stylistic thing, but I'd probably separate these out to make it a little easier to read:

```elixir
rendered = render_template(command, adapter, template, data)
{:cont, [rendered|acc]}
```